model/AdditiveManufacturing/Opc.Ua.AdditiveManufacturing.Nodeset2.xml

```diff
@@ -0,0 +1,1339 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UANodeSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:uax="http://opcfoundation.org/UA/2008/02/Types.xsd" xmlns="http://opcfoundation.org/UA/2011/03/UANodeSet.xsd" xmlns:s1="http://yourorganisation.org/Opc.Ua.AdditiveManufacturing.NodeSet2/Types.xsd" xmlns:s2="http://opcfoundation.org/UA/MachineTool/Types.xsd" xmlns:s3="http://opcfoundation.org/UA/Dictionary/IRDI/Types.xsd" xmlns:ua="http://unifiedautomation.com/Configuration/NodeSet.xsd" xmlns:s4="http://opcfoundation.org/UA/DI/Types.xsd" xmlns:s5="http://opcfoundation.org/UA/Machinery/Types.xsd" xmlns:s6="http://opcfoundation.org/UA/Machinery/ProcessValues/Types.xsd" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <NamespaceUris>
+        <Uri>http://yourorganisation.org/Opc.Ua.AdditiveManufacturing.NodeSet2/</Uri>
```

Comment:
This does not look right. In the constant files, the namespace is named: `http://opcfoundation.org/AdditiveManufacturing/`

I would guess that the final URI might be `http://opcfoundation.org/UA/AdditiveManufacturing/` or `http://opcfoundation.org/UA/AM/`